packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-editor.tsx

```diff
@@ -34,15 +34,14 @@ const containerStyles = css({
 
 const editorContainerStyles = css({
   flex: '1 1 100%',
-  overflow: 'auto',
+  overflow: 'hidden',
 });
 
 const errorContainerStyles = css({
   flex: 'none',
   marginTop: 'auto',
   marginLeft: spacing[3],
   marginRight: spacing[3],
-  marginBottom: spacing[2],
```

Comment:
@mabaasit any reason we returned this bottom margin and the one below? I removed them in a PR about a week ago because they make bottom content misaligned (and make it hard to move content around in the editor because they clash with the paddings of the container):

<img width="700" alt="image" src="https://user-images.githubusercontent.com/5036933/203402645-fe43662a-95f5-4bab-a8cd-8cb41238da65.png">
<img width="700" alt="image" src="https://user-images.githubusercontent.com/5036933/203402679-27e11d3c-e95a-4479-877e-d1b0518a5f7e.png">

Reply:
it was a mistake to do that. the changes you've made in this PR regarding spacing fixes it.

Reply:
Cool, thanks! Just double-checking I am not missing some corner-case state that this was accounting for